packages/features/insights/components/AverageEventDurationChart.tsx

```diff
@@ -12,7 +12,7 @@ import { LoadingInsight } from "./LoadingInsights";
 export const AverageEventDurationChart = () => {
   const { t } = useLocale();
   const { filter } = useFilterContext();
-  const { dateRange, selectedMemberUserId } = filter;
+  const { dateRange, selectedMemberUserId, isOrg } = filter;
```

Comment:
new prop in UI that tell us if Team option selected is the team that is parent of other teams.